text/0021-internal-data-monitoring.md

```diff
@@ -0,0 +1,182 @@
+---
+rfc pr: https://github.com/UCLA-Creative-Labs/cl-rfcs/pull/22
+tracking issue: https://github.com/UCLA-Creative-Labs/cl-rfcs/issues/21
+---
+
+# Watchdog
+
+`Watchdog` is a monorepo that will contain a variety of Creative Labs
+data analysis tools for internal use. 
+
+## Blurb
+
+The goal of this project is to allow all the teams across Creative Labs
+board to make data-driven decisions. We want to set up a system where all
+members can access previous data and use insights from the past to make
+good judgement calls in the present.
+
+Watchdog is meant to be a monorepo to house various data analysis tools
+for Creative Labs. There are a few ideas to implement currently. These
+projects are primarily meant for Creative Labs board members, but we
+could look towards displaying some of the data publicly or publishing
+our findings through a blog post for marketing content!
+
+Note: Since we do not have complete datasets on many events or projects in
+the past, much of this will be just to setup infrastructure for the future.
+
+### Project 1: Project Data Vis
```

Comment:
I would like it if you could expand on this further!

Here are some things that are important for you to cover:
* Are we going to restrict access to this web app? If so, how do we accomplish that
* How are we planning on building this web app? React? Next.js? Server Side React?
* Where are we going to store all these data sets?
* What will the data model look like and how can we easily grab these values?

Reply:
i dont believe this is resolved